sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/UcfgGeneratorTest.cs

```diff
@@ -169,6 +170,38 @@ public void UcfgGenerator_Disabled_NoOutputFolder()
             UcfgGenerator_Disabled(null, "S3649");
         }
 
+
+        [TestMethod]
+        public void UcfgGenerator_Enabled_Generate_NoErrorForUnsupportedLanguageConstructs()
+        {
+            var workDir = Path.Combine(TestContext.TestRunResultsDirectory, TestContext.TestName, "0");
+            Directory.CreateDirectory(workDir);
+
+            using (new AssertIgnoreScope())
```

Comment:
Might be worth to document which kind of assertion is broken/triggered here (i.e. why do we have this `AssertIgnoreScope`?)